src/www/.storybook/localize.ts

```diff
@@ -0,0 +1,45 @@
+/*
+  Copyright 2022 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import englishMessages from "../messages/en.json";
```

Comment:
Can we pass url parameters to storybook? Perhaps that could be a way to pass the language. But not for this PR.

Reply:
We'd need an addon to support that, unfortunately